application/Commands/Layouts/Export.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace OTGS\Toolset\CLI\Layouts;
+
+use WPDD_Layouts_Theme ;
+use ZIP ;
+
+/*
+ * FIXME: replace \WP_CLI with WpCli.
+ */
+
+/**
+ * Layouts export command.
+ */
+class LayoutsExport extends LayoutsCommand {
+
+	/**
+	 * Exports Layouts to a ZIP file.
+	 *
+	 * ## Options
+	 *
+	 * [--overwrite]
+	 * : Overwrite existing ZIP file.
+	 *
+	 * <file>
+	 * : The path of the exported ZIP file.
+	 *
+	 * ## Examples
+	 *
+	 *     wp layouts export <file>
+	 *
+	 * @param array $args The array of command-line arguments.
+	 * @param array $assoc_args The associative array of command-line options.
+	 */
+
+	public function __invoke( $args, $assoc_args ) {
+
+		list( $export_filename ) = $args;
+
+		if ( empty ( $export_filename ) ) {
+			\WP_CLI::warning( __( 'You must specify a valid file.', 'toolset-cli' ) );
```

Comment:
This seems more like an error, IMHO. 

Also, an example of how you can use the `WpCli` class that is a wrapper for interactions with the WP CLI core:

```suggestion
			$this->wp_cli()->error( __( 'You must specify a valid file.', 'toolset-cli' ), true );
```